### PR TITLE
UI uploadlstwords refinements

### PR DESCRIPTION
**Pull request by davidfortuoso-ti**
Now the create list of words page allows deleting unwanted entries

image

### CODE REVIEW

File: src/pages/postlstwords.tsx [hunk 1]
```diff
@@ -332,27 +338,42 @@ const CreateListOfWords: NextPage = () => {
...
-      } else if (value === "3") {
-        handleRecommeddedStoryList();
-        setStoryGenerated(true);
-        setAcronymGenerated(false);
-      }
-      setTableEntryValue("");
-    } else {
-      throw new Error("Missing element 'answer' table");
+    let item = {
+      id: uuidv4(),
```
Comment (gao-ti) [inline src/pages/postlstwords.tsx:342]: do we have to use uuid and v4?
It drags down the performance and introduces some vulnerabilities afaik
Do we have other unique things that can be used as id?

File: src/pages/postlstwords.tsx [hunk 2]
```diff
@@ -332,27 +338,42 @@ const CreateListOfWords: NextPage = () => {
...
+    if (index > -1) {
+      options.splice(index, 1);
+      wordList.splice(index, 1);
+      setOptions((state) => [...state]);
+    }
+    if (value === "1") {
+      handleRecommeddedAcronymList();
+      setAcronymGenerated(true);
+      setStoryGenerated(false);
+    } else if (value === "3") {
```
Comment (gao-ti) [inline src/pages/postlstwords.tsx:371]: make enum for 1, 2, 3 to make it understandable

File: src/pages/postlstwords.tsx [hunk 3]
```diff
@@ -480,8 +502,37 @@ const CreateListOfWords: NextPage = () => {
...
+                                border: "1px solid #ccc",
+                                borderRadius: 5,
+                                marginBottom: 5,
+                              }}
+                            >
+                              <p>{option.title}</p>
+                              <button
+                                onClick={() => {
+                                  handleDelete(option.id);
+                                  //handleRemoveFromSequence(index);
```
Comment (gao-ti) [inline src/pages/postlstwords.tsx:525]: remove comment

File: src/pages/postlstwords.tsx (hunk 1 above)
Comment (davidfortuoso-ti) [reply src/pages/postlstwords.tsx]: Switched from uuidv4 to nanoid on [3aa06e7](https://github.com/trilogy-group/memorization-app/pull/165/commits/3aa06e72e6d0a3efcb7ae67613501798709296b0)

File: src/pages/postlstwords.tsx (hunk 3 above)
Comment (davidfortuoso-ti) [reply src/pages/postlstwords.tsx]: [7ea2b8f](https://github.com/trilogy-group/memorization-app/pull/165/commits/7ea2b8f61a1b8a18daf7859e616a240982f33d92)

File: src/pages/postlstwords.tsx (hunk 2 above)
Comment (davidfortuoso-ti) [reply src/pages/postlstwords.tsx:368]: I don't get this

Comment (davidfortuoso-ti) [reply src/pages/postlstwords.tsx]: [363ac36](https://github.com/trilogy-group/memorization-app/pull/165/commits/363ac36bea2c6fd8d7f9767e1e9177e07a2698ff)

**Pull request merged**